server/controllers/finance/creditors.js

```diff
@@ -63,3 +66,82 @@ function detail(req, res, next) {
     .catch(next)
     .done();
 }
+
+/**
```

Comment:
In the long run, we should probably combine the `debtors.js` and `creditors.js` files into an `entities.js` file.  I think this code is almost identical to the debtor code.